chain/network/src/types.rs

```diff
@@ -162,6 +162,9 @@ pub struct SetChainInfo(pub ChainInfo);
 #[rtype(result = "PeerManagerMessageResponse")]
 pub enum PeerManagerMessageRequest {
     NetworkRequests(NetworkRequests),
+    /// Request PeerManager to advertise tier 1 proxies.
+    /// Used internally.
+    AdvertiseTier1Proxies,
```

Comment:
can you add more comment here about what advertising the proxy means? that is the effect of this?

Reply:
Done https://github.com/near/nearcore/pull/11536/commits/43af2326c0c15de0a5cc7730a5dc4e0ad4552dc1